packages/webapp/src/components/Task/FieldWorkTask/index.jsx

```diff
@@ -41,6 +36,31 @@ const PureFieldWorkTask = ({
   useEffect(() => {
     if (fieldWorkTypeExposedValue?.value !== 'OTHER') setValue(FIELD_WORK_OTHER_TYPE, null);
   }, [fieldWorkTypeExposedValue]);
+
+  useEffect(() => {
```

Comment:
use useMemo and remove useState setState

Reply:
const fieldWorkTypeOptions = useMemo(()=>{
let options = ....
return options
}, [fieldWorkTypes])